### PR TITLE
Removed conditional code for no longer supported versions of CMake. Unified code style.

### PR DESCRIPTION
**Pull request by SunBlack**
See commit messages ;-)

### CODE REVIEW

File: cmake/pcl_find_sse.cmake [hunk 1]
```diff
@@ -62,7 +62,7 @@ macro(PCL_CHECK_FOR_SSE)
           __m128i vc = _mm_cmpgt_epi64 (va, vb);
 
           _mm_storeu_si128 ((__m128i*)c, vc);
-          if (c[0] == -1LL && c[1] == 0LL)
+          if(c[0] == -1LL && c[1] == 0LL)
```
Comment (taketwo) [inline cmake/pcl_find_sse.cmake]: This is not _cmake_ code ;)

File: cmake/pcl_targets.cmake [hunk 2]
```diff
@@ -101,28 +101,28 @@ macro(PCL_SUBSYS_DEPEND _var _name)
...
+        endif()
         if(SUBSYS_EXT_DEPS)
         foreach(_dep ${SUBSYS_EXT_DEPS})
             string(TOUPPER "${_dep}_found" EXT_DEP_FOUND)
             if(NOT ${EXT_DEP_FOUND} AND (NOT (${EXT_DEP_FOUND} STREQUAL "TRUE")))
                 set(${_var} FALSE)
                 PCL_SET_SUBSYS_STATUS(${_name} FALSE "Requires external library ${_dep}.")
-            endif(NOT ${EXT_DEP_FOUND} AND (NOT (${EXT_DEP_FOUND} STREQUAL "TRUE")))
+            endif()
         endforeach(_dep)
```
Comment (taketwo) [inline cmake/pcl_targets.cmake]: You can apply the same fix for `foreach` loops.

File: cmake/uninstall_target.cmake.in [hunk 3]
```diff
@@ -17,7 +17,7 @@ foreach(file ${files})
   endif()
 endforeach()
 
-# remove pcl directory in include (removes all files in it!)
+# remove pcl directory in include(removes all files in it!)
```
Comment (taketwo) [inline cmake/uninstall_target.cmake.in]: Please undo this

**Comment by SergioRAgostinho** [comment]
@SunBlack Did you use a formatting tool? Which one?

**Comment by SunBlack** [comment]
Notepad++ => Regex Search & Replace ;-)

**Comment by taketwo** [comment]
I think this is generated by a sequence of search/replace operations with regex :)

**Comment by SergioRAgostinho** [comment]
This came to mind
<a href=https://xkcd.com/208/></a>

File: cmake/pcl_find_sse.cmake (hunk 1 above)
Comment (SunBlack) [reply cmake/pcl_find_sse.cmake]: You are right, I forgot that some files are not pure CMake ;-)

File: cmake/uninstall_target.cmake.in (hunk 3 above)
Comment (SunBlack) [reply cmake/uninstall_target.cmake.in]: Done

File: cmake/pcl_targets.cmake (hunk 2 above)
Comment (SunBlack) [reply cmake/pcl_targets.cmake]: Done

**Comment by SergioRAgostinho** [comment]
Not sure what to do here. Our CI pipeline for windows broke today and I would like to ensure that things on Windows are also running ok, just for precaution.

**Comment by SunBlack** [comment]
Thanks @jasjuang. I will try this tool later. I think we have enough changes in this PR for now.

**Pull request merged**